jjb/jobs/projects.yaml

```diff
@@ -21,8 +21,13 @@
         - 'master'
         - 'PILOT'
     distro:
-        - 'f26'
-        - 'f27'
-        - 'rhel7'
-        - 'rhel6'
-        - 'centos7'
+        - 'f26':
```

Comment:
You are repeating the same repo-file three times you can move it to the project definition and customize it on rhel7 and rhel6. The `rho-copr-install` uses this to avoid repeating epel-7 as the repo.